### ISSUE TITLE
Room booking: add a help button in the list of rooms page

### ISSUE THREAD

**Issue by jouvin** [indico/indico#4063]
Despite the page displaying the list of rooms is well designed, new users (at least at LAL) seem to have problems to identify how to do actions like selecting a room, filtering the list... May be, one thing that could be helpful for these new users without cluttering the page for the others, would be to have a small help button at the top of the page that would display a text box (like when the cursor is over a button) explaining the buttons that show up when you clicked on a room, the possibility to filter the room, the possibility to navigate the map and select a room from it...

**Comment by ThiefMaster** [comment]
There's a link to the documentation in the side menu

**Comment by jouvin** [comment]
Yes, but I don't think it is really fullfiling the same need. A "standard user" who uses the booking for the first time or only occasionally doesn't want to be redirected to a full documentation, in addition in English... Thus my proposal to have, only in page listing all the rooms as I think it is the only one with some "complexity" and not-so-visible advanced features, that will display a short help on this page features.
